Automattic/jetpack
# Jetpack: revert psr/container package update

#### Changes proposed in this Pull Request:

Revert dependency updates done in #24334

Otherwise when activating Jetpack + WooCommerce plugins on local JP dev a fatal is thrown:

```
Fatal error:  Declaration of Automattic\WooCommerce\Container::has($id) must be compatible with Psr\Container\ContainerInterface::has(string $id): bool in /var/www/html/wp-content/plugins/woocommerce/src/Container.php on line 102
```

#### Jetpack product discussion

- Internal: p1654886291453829-slack-CDLH4C1UZ
- Related: #17221

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- Pull down changes, `composer install` / build Jetpack.
- Ensure that you are able to run Jetpack + Woo plugins at same time with no error.
